RetrospectiveExtension.Backend/deploy/env_setup.sh

```diff
@@ -79,6 +79,21 @@
         --query instrumentationKey \
         --output tsv)
 
+    # Add an Application Insights dashboard
+    dashboard_file="./deploy/ai-dashboard.json"
+
+    cp ./deploy/ai-dashboard.json.template ${dashboard_file}
+  
+    sed -i "s/VARRGNAME/${resource_group}/"                     ${dashboard_file}
```

Comment:
```suggestion
    sed -i "" "s/VARRGNAME/${resource_group}/" ${dashboard_file}
```
On macOS this command will fail since a suffix is required. Using a blank suffix will keep the same behavior.